package com.glitch.securenotes.data.exceptions.auth

class CredentialsNotFoundException(): Throwable("Credentials for that user is unavailable")